pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "crdch_example_workflows"
+version = "0.1.0"
+description = "Example workflows and data for the CRDCH model"
+authors = ["Gaurav Vaidya <gaurav@renci.org>"]
+
+[tool.poetry.dependencies]
+python = ">=3.7.1,<4.0"
+linkml = "^1.1.13"
```

Comment:
The discrepancy in dependency versions used in ccdhmodel and example-data repos.

* The python version on ccdhmodel on [pyproject.toml](https://github.com/cancerDHC/ccdhmodel/blob/switch-to-poetry/pyproject.toml) (on switch-to-potery branch) is `python = ">=3.7,<4.0"`. 
* Which linkml / linkml-runtime version do we want to use? Do we want to use the packages from PyPI only on example-data repo?

Reply:
Good catch! I've fixed the Python version issue in a5de29d.

Re: LinkML, I think we generally want to use the most recent version. This version number should change when we run `poetry update`, so it should eventually change to e.g. 1.2.* once LinkML shifts to that version. I think we can leave it as "^1.1.13", i.e. "something compatible with the latest version as of the time this library was developed".

(For the kind of stuff we do in this repo, the LinkML-Runtime version is probably more important than the LinkML version, but that's imported by crdch_model as a dependency.)